commit 0e222e9a1ab7b421013b02e3b458c44e00e84bff (refs/changes/46/546/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2016-09-29T00:58:53+00:00 (3 years ago)
    
    joyent/node-manta#275 msign with subusers broken
    Reviewed by: David Pacheco <dap@joyent.com>
    Reviewed by: Brian Bennett <brian.bennett@joyent.com>

diff --git a/bin/msign b/bin/msign
index e44805e..6e3d490 100755
--- a/bin/msign
+++ b/bin/msign
@@ -135,8 +135,7 @@ function parseOptions() {
             path: p,
             sign: client.sign,
             user: opts.user,
-            subuser: opts.subuser,
-            mantaSubUser: true
+            subuser: opts.subuser
         };
         client.signURL(_opts, function (err, resource) {
             ifError(err);
diff --git a/lib/client.js b/lib/client.js
index f1e4485..4fe45e2 100644
--- a/lib/client.js
+++ b/lib/client.js
@@ -3056,6 +3056,7 @@ MantaClient.prototype.signURL = function signURL(opts, cb) {
         sign: opts.sign || self.sign,
         user: opts.user || self.sign.user || self.user,
         subuser: opts.subuser || self.sign.subuser || self.subuser,
+        mantaSubUser: true,
         manta: true
     };
 
diff --git a/package.json b/package.json
index e3e7c4c..5af519b 100644
--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
         "type": "git",
         "url": "git://github.com/joyent/node-manta.git"
     },
-    "version": "3.1.1",
+    "version": "3.1.2",
     "main": "./lib/index.js",
     "dependencies": {
         "assert-plus": "^1.0.0",
